libs/design/src/core/storage/server-safe-persistence.token.ts

```diff
@@ -0,0 +1,22 @@
+import { isPlatformBrowser } from '@angular/common';
+import {
+  InjectionToken,
+  inject,
+  PLATFORM_ID,
+} from '@angular/core';
+
+import {
+  DaffPersistenceService,
+  DaffPersistenceServiceToken,
+  DaffNoopStorageService,
+} from '@daffodil/core';
+
+export const DaffServerSafePersistenceServiceToken = new InjectionToken<
```

Comment:
This looks like something that belongs in `@daffodi/core/storage`.

Reply:
moved